nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/RuntimeHarness.java

```diff
@@ -120,6 +120,10 @@
      */
     RuntimeContext deployPartial(String bundle, Set<TargetExtensions> targetExtensions) throws Exception;
 
+    /**
+     * @deprecated since 10.2
```

Comment:
Why ? what should we use instead ?

Reply:
It's not used, badly implemented, thought it was a good thing to tell people not to use this anymore.

Reply:
Ok then you can say `@deprecated since 10.2, unused and badly implemented`
Same of other `@deprecated` methods in this class, always try to explain why